gatsby-config.js

```diff
@@ -61,13 +61,13 @@ module.exports = {
 				},
 			},
 		},
-		{
+		/* 	{
```

Comment:
Maybe write a comment why google analytics is commented out (cookies and migration to google analytics 4)